src/Images.jsx

```diff
@@ -141,11 +141,11 @@ class Images extends React.Component {
 
         const columns = [
             { title: utils.image_name(image), header: true, props: { modifier: "breakWord" } },
-            { title: (image.uid == 0) ? _("system") : <div><span className="ct-grey-text">{_("user:")} </span>{user.name}</div>, props: { className: "ignore-pixels", modifier: "nowrap" } },
-            { title: <utils.RelativeTime time={image.Created * 1000} />, props: { className: "image-created" } },
+            { title: (image.uid == 0) ? _("system") : <div><span className="ct-grey-text">{_("user:")} </span>{user.name}</div>, props: { className: "ignore-pixels", modifier: "nowrap" }, rawValue: user.name },
+            { title: <utils.RelativeTime time={image.Created * 1000} />, props: { className: "image-created" }, rawValue: image.Created },
             { title: utils.truncate_id(image.Id), props: { className: "image-id" } },
-            { title: cockpit.format_bytes(image.Size), props: { className: "ignore-pixels", modifier: "nowrap" } },
-            { title: <span className={usedByCount === 0 ? "ct-grey-text" : ""}>{usedByText}</span>, props: { className: "ignore-pixels", modifier: "nowrap" } },
+            { title: cockpit.format_bytes(image.Size), props: { className: "ignore-pixels", modifier: "nowrap" }, rawValue: image.Size },
+            { title: <span className={usedByCount === 0 ? "ct-grey-text" : ""}>{usedByText}</span>, props: { className: "ignore-pixels", modifier: "nowrap" }, rawValue: usedByCount },
```

Comment:
Please rename these to `sortKey`, thats what the conditional is on line `239` and what we use everywhere in Cockpit for this purpose.